keyboards/s7_elephant/keymaps/default/keymap.c

```diff
@@ -0,0 +1,95 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```

Comment:
This isn't actually needed and could be removed. 
```suggestion
```

Reply:
Fixed.